apis/python/tests/test_fastercsx.py

```diff
@@ -77,7 +78,7 @@ def assert_eq(sp: sparse.spmatrix, cm: fastercsx.CompressedMatrix) -> bool:
 def test_from_ijd(
     shape: tuple[int, int],
     value_dtype: np.dtype[Any],
-    format: Literal["csc", "csr"],
+    format: Format,
```

Comment:
In future this kind of thing could be split out as a separate PR

Reply:
Done: #3770